node/thermostat/main.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+const os = require('os')
+
+const DigitalTwinClient = require('azure-iot-digitaltwins-device').DigitalTwinClient
+
+const DeviceInformation = require('./deviceInformation').DeviceInformation
+const SdkInformation = require('./sdkInformation').SdkInformation
+const TemperatureSensor = require('./temperatureSensor').TemperatureSensor
+const Diagnostics = require('./diagnostics').Diagnostics
+
+const modelId = 'dtmi:com:example:Thermostat;1'
```

Comment:
>modelId [](start = 6, length = 7)

You don't include a model file in your example.

Reply:
the model is in the target branch. Mode First :)